### ISSUE TITLE
Image uploader resize dropdown confusing

### ISSUE THREAD

**Issue by anthonyisensee** [HoltBosse/Alba#257]
![image](https://user-images.githubusercontent.com/60198052/214148627-68caac88-7c87-40ad-bf72-35ab57fb3b84.png)

If I select "Yes" for Web Friendly, should that mean the image I'm uploading is web friendly? Or that the image should be resized for the web? Either way, this is confusing and sends mixed messages.

I would suggest we change the select dropdown label to read "Resize" and modify the p.help to read "Resizes larger images for the web.

![image](https://user-images.githubusercontent.com/60198052/214149815-f3d999b2-95cc-4890-8bc1-826b53d1c169.png)

**Comment by workaj** [comment]
`web friendly` makes more sense for non technical users, granted, it could use something like `make` in front of it

**Comment by workaj** [comment]
since we talked about in the past and no one is yet to do it, is make alt just be what title is, you could possibly just have image title with a details element saying additional options containing alt override and "web friendly since no one really uses them"

**Comment by anthonyisensee** [comment]
Right, but does "Web Friendly" imply that the image is ALREADY web friendly, or needs to be MADE web friendly by the CMS? See what I'm saying? It's ambiguous enough that I genuinely don't know which option does which.

**Comment by workaj** [comment]
![image](https://user-images.githubusercontent.com/109080500/214153731-1f713474-e4a6-4029-a3b6-b2dc1d8c4c88.png)
very crappy edit, where the items in the box would be hidden by default (since they almost never get used/changed from default)

**Comment by anthonyisensee** [comment]
Definitely more straightforward!

So we can find it later:

https://github.com/HoltBosse/Alba/blob/c8d824767ad065774adabd736a97037e256c4b62/admin/controllers/images/views/show/script.js#L415
&
https://github.com/HoltBosse/Alba/blob/c8d824767ad065774adabd736a97037e256c4b62/admin/controllers/images/views/show/script.js#L420
